docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +61,162 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
+         - class: default-worker
+           metadata: {}
+           replicas: 3
+           name: md-0
++        - class: default-worker
++          metadata: {}
++          replicas: 1
++          name: second-deployment
+```
+
+After a minute to scale the new MachineDeployment we get:
+```bash
+NAME                                      CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+capi-quickstart-md-0-XXXX                 capi-quickstart   1          1       1         0             Running   39m   v1.23.3
+capi-quickstart-second-deployment-XXXX    capi-quickstart   1          1       1         0             Running   99s   v1.23.3
+```
+Our second deployment uses the same underlying MachineDeployment class `default-worker` as our initial deployment. In this case they will both have exactly the same underlying machine templates. In order to modify the templates MachineDeployments are based on take a look at [Changing a ClusterClass].
+
+A similar process described here - removing the machineDeployment from `cluster.spec.topology.workers.machineDeployments` - can be used to delete a running MachineDeployment from an active Cluster.
+
+## Scale a ControlPlane
+When using a managed topology scaling of ControlPlane Machines, where the Cluster is using a topology that includes ControlPlane MachineInfrastructure, should be done through the Cluster topology.
+
+This is done by changing the ControlPlane replicase field at `/spec/topology/controlPlane/replica` in the Cluster object. The command is:
+
+```bash 
+kubectl  patch cluster capi-quickstart --type json --patch '[{"op": "replace", "path": "/spec/topology/controlPlane/replicas",  "value": 1}]'
+```
+
+This patch will make the below changes on the Cluster yaml:
+```diff
+   spec:
+      topology:
+        controlPlane:
+          metadata: {}
++         replicas: 1
+-         replicas: 3
+```
+
+As well as scaling a ControlPlane, Cluster authors can edit the labels and annotations applied to a running ControlPlane using the Cluster topology as a single point of control.
+
+
+## Use variables
+A ClusterClass can use variables and patches in order to allow flexible customization of Clusters derived from a ClusterClass. Variable definition allows two or more Cluster topologies derived from the same ClusterClass to have different specs, with the differences defined in variables in the Cluster topology.
+
+Assume we have created a CAPD cluster with ClusterClass and Kubernetes v1.23.3 (as documented in the [Quick Start guide]). Our Cluster has a variable `etcdImageTag` as defined in the ClusterClass. The variable is not set our Cluster. In this case `etcdImageTag` is using a default value defined in the ClusterClass but not all variables will have a default value defined. Some variables, depending on their definition in a ClusterClass, may need to be specified by the Cluster author for every Cluster created using a given ClusterClass..
```

Comment:
```suggestion
Assume we have created a CAPD cluster with ClusterClass and Kubernetes v1.23.3 (as documented in the [Quick Start guide]). Our Cluster has a variable `etcdImageTag` as defined in the ClusterClass which is currently not set. In this case `etcdImageTag` is using a default value defined in the ClusterClass but not all variables will have a default value defined. Some variables, depending on their definition in a ClusterClass, may need to be specified by the Cluster author for every Cluster created using a given ClusterClass.
```
(please note the double trailing dot at the end)

Reply:
Hm, it's not really correct that the default value is used. The variable etcdImageTag is set to an empty string (and the default would be an empty string too, but is not used).
Maybe we should remove that remark about defaults?